src/apps/lwaftr/lwaftr.lua

```diff
@@ -599,6 +581,21 @@ function LwAftr:push ()
    local o4, o6 = self.output.v4, self.output.v6
    self.o4, self.o6 = o4, o6
 
+   do
+      local msg = self.control:pop()
+      if msg then
+         if msg == messages.lwaftr_message_reload then 
```

Comment:
so, this is bogus of course -- we need to be checking msg.kind == messages.lwaftr_message_reload